pelican/__init__.py

```diff
@@ -73,6 +73,7 @@ def init_plugins(self):
             except Exception as e:
                 logger.error('Cannot register plugin `%s`\n%s',
                              plugin.__name__, e)
+        stringify_plugins(self.settings)
```

Comment:
It is probably better to stringify loaded plugins (`self.plugins` or subset if some failed) instead of the original `PLUGINS` setting.

Reply:
Would updating `self.plugins` also update the reference in the `settings` module, thus allowing us to pickle it?